tmk_core/common/avr/suspend.c

```diff
@@ -111,7 +111,6 @@ static void power_down(uint8_t wdto) {
         rgblight_disable_noeeprom();
     }
 #    endif
-    suspend_power_down_kb();
```

Comment:
This should be left here?

Reply:
No; it is also called in `suspend_power_down()`, which calls `power_down()`. With the call in both places, it is called twice per cycle, which I believe is not needed.